code/modules/projectiles/gun.dm

```diff
@@ -331,83 +345,81 @@
 #undef FIREARM_MESSAGE_SPAM_TIME
 
 //called if there was no projectile to shoot
-/obj/item/gun/proc/handle_click_empty(mob/user)
+/obj/item/gun/proc/handle_click_empty(atom/movable/firer)
 	if(check_fire_message_spam("click"))
-		if(user)
-			user.visible_message("*click click*", "<span class='danger'>*click*</span>")
+		if(isliving(firer))
```

Comment:
Does this need to be living specifically? I noticed during the video you posted turrets click empty and make the noise, so the message seems appropriate.